### PR TITLE
Imagelink

### PR DESCRIPTION
**Pull request by allancodenet**
## Pull request checklist

- [x] My code contains tests covering the code I modified
- [x] I linted and tested the project with `bin/check`
- [ ] I added significant changes and product updates to the [changelog](CHANGELOG.md)

### CODE REVIEW

**Comment by joemasilotti** [comment]
Thanks for the PR, @allancodenet! I'm a bit confused though - why does adding a link increase accessibility? There is already an alt tag for screen readers.

**Comment by allancodenet** [comment]
I see that now, It sis not a very necessary feature but It still encourages the action  to go to your personal site

File: db/schema.rb [hunk 1]
```diff
@@ -328,7 +327,7 @@
     t.string "processor_id"
     t.boolean "default"
     t.jsonb "data"
-    t.datetime "deleted_at", precision: nil
```
Comment (nunommc) [inline db/schema.rb:331]: surely the db/schema doesn't need to change

**Comment by joemasilotti** [comment]
> I see that now, It is not a very necessary feature but It still encourages the action to go to your personal site

That's true but there is already a link on the page. I'd rather not spam people with it!

**Pull request closed without merging**